nautobot_circuit_maintenance/urls.py

```diff
@@ -3,33 +3,19 @@
 from django.templatetags.static import static
 from django.urls import path
 from django.views.generic import RedirectView
+from nautobot.core.views.routers import NautobotUIViewSetRouter
 from nautobot.extras.views import ObjectChangeLogView
 
 from . import views
-from .models import CircuitImpact, CircuitMaintenance, Note, NotificationSource
+from .models import CircuitImpact, NotificationSource
+
+router = NautobotUIViewSetRouter()
+router.register("maintenance", views.CircuitMaintenanceUIViewSet)
```

Comment:
This is going to collide heavily with #320 isn't it?

Reply:
yes, I plan to converge to the #320 one